src/slack/constants.bal

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+const string BASE_URL = "https://slack.com/api";
+const string CREATE_CONVERSATION_PATH = "/conversations.create?name=";
```

Comment:
Instead of making the entire path like above a constant, shall we name the constants meaningfully.
For example,
```
const string CONVERSATION_PATH = "/conversations";
const string OPERATION_CREATE = "create";
const string QUERY_PARAM_NAME = "name";
```
And can we re-organize the other constants as well.

Reply:
I think the existing way is more readable.